Purchases/Public/RCPurchases.m

```diff
@@ -1011,14 +1010,14 @@ - (void)receiptData:(RCReceiveReceiptDataBlock)completion {
 - (void)receiptDataWithReceiptRefreshPolicy:(RCReceiptRefreshPolicy)refreshPolicy
                                  completion:(RCReceiveReceiptDataBlock)completion {
     if (refreshPolicy == RCReceiptRefreshPolicyAlways) {
-        RCDebugLog(@"Forced receipt refresh");
+        RCDebugLog(@"%@", RCStrings.receipt.force_refreshing_receipt);
```

Comment:
could this be 
```objective-c
RCDebugLog(RCStrings.receipt.force_refreshing_receipt);
```
?

Reply:
@aboedo similar to comment in #426, Xcode complains when there's no `@"%@"`